### PR TITLE
Stabilize the rust-version field

### PR DESCRIPTION
**Pull request by djc**
I've tried to make the documentation here fairly comprehensive. I've also updated the first version for the 2021 edition, which should now be stable pending substantial unforeseen changes.

See #8072.

### CODE REVIEW

**Comment by ehuss** [comment]
Thanks!  Normally the documentation is updated along with the stabilization.  Are you intending to add more commits here to stabilize?

As for the 2021 stabilization, there still is some risk it could slip a release, since 6 weeks isn't much time.  Hopefully it won't, but I assume changing the value shouldn't cause too much of a problem since 2021 is still unstable?

**Comment by djc** [comment]
Ah, wasn't sure about the exact wording you used in the tracking issue. I'll add a commit to stabilize.

It will be easy to update the first version for 2021 again if necessary. We can do that if/once 1.56 hits beta without 2021.

**Comment by djc** [comment]
Add a commit to stabilize.

**Comment by joshtriplett** [comment]
Stabilizing this seems reasonable to me.

@rfcbot merge

**Comment by joshtriplett** [comment]
Stabilizing this seems reasonable to me.

@rfcbot merge

File: src/cargo/core/features.rs [hunk 1]
```diff
@@ -395,9 +394,6 @@ features! {
     // Allow to specify whether binaries should be stripped.
     (unstable, strip, "", "reference/unstable.html#profile-strip-option"),
 
-    // Specifying a minimal 'rust-version' attribute for crates
-    (unstable, rust_version, "", "reference/unstable.html#rust-version"),
```
Comment (ehuss) [inline src/cargo/core/features.rs:399]: This should be marked as "stable", see the other entries for the format.

File: src/doc/src/reference/unstable.md [hunk 2]
```diff
@@ -1170,25 +1169,6 @@ cargo logout -Z credential-process
 [crates.io]: https://crates.io/
 [config file]: config.md
 
-### rust-version
```
Comment (ehuss) [inline src/doc/src/reference/unstable.md:1173]: Can you add a section at the bottom to indicate that this is now stable (see the other entries for examples).

**Review by ehuss — COMMENTED** [review]
`arg_ignore_rust_version` needs to remove the `(unstable)` in the help.

`--ignore-rust-version` needs to be added to the man pages.  There is some documentation at https://github.com/rust-lang/cargo/tree/master/src/doc on how to modify those.  I would probably put it in the `Compilation Options` section. There should be plenty of examples of other options on how to share across the man pages.

File: src/cargo/core/features.rs (hunk 1 above)
Comment (djc) [reply src/cargo/core/features.rs:399]: Fixed.

File: src/doc/src/reference/unstable.md (hunk 2 above)
Comment (djc) [reply src/doc/src/reference/unstable.md:1173]: Done.

**Comment by ehuss** [comment]
Thanks!

@bors r+

**Comment by djc** [comment]
@ehuss thank you for all the reviews and mentoring, and your awesome work in Cargo in general!

**Comment by djc** [comment]
Is it just me, or was the FCP cut short for this issue? Shouldn't the FCP have happened in the tracking issue? Very confused about the process now.

**Comment by ehuss** [comment]
We mostly use rfcbot just to get a positive approval from the entire team.  We don't broadcast these like the lang team does, and don't usually wait for the comment period.  Of course, anyone is welcome to leave comments at any time, and there is 6 to 12 weeks to change or revert any decision.

**Pull request merged**